tests/perfalert/test_alerts/test_backfill_report_maintainer.py

```diff
@@ -0,0 +1,216 @@
+import datetime
+import random
+import string
+from typing import Tuple
+from unittest.mock import Mock
+
+import pytest
+
+from treeherder.model.models import (MachinePlatform,
+                                     Option,
+                                     OptionCollection)
+from treeherder.perf.alerts import (AlertsPicker,
+                                    BackfillReportMaintainer)
+from treeherder.perf.models import (BackfillRecord,
+                                    BackfillReport,
+                                    PerformanceAlert,
+                                    PerformanceSignature)
+
+LETTERS = string.ascii_lowercase
+EPOCH = datetime.datetime.utcfromtimestamp(0)
+RANDOM_STRINGS = set()
+
+
+@pytest.fixture(scope='module')
```

Comment:
Could we move fixtures into a conftest.py file?

Reply:
I would avoid doing that, as the `conftest.py` is already very bloated. These fixtures can't really be reused elsewhere. Once this is noticed, some of the fixtures here can be elevated in `conftest.py`.